Packet++/header/TcpReassembly.h

```diff
@@ -89,9 +89,9 @@ struct ConnectionData
 	/** A 4-byte hash key representing the connection */
 	uint32_t flowKey;
 	/** Start TimeStamp of the connection */
-	timeval startTime;
+	timespec startTime;
 	/** End TimeStamp of the connection */
-	timeval endTime;
+	timespec endTime;
```

Comment:
I'm afraid this might be a breaking change for everyone using `ConnectionData`. Maybe we should keep `startTime` and `endTime` as `timeval` , add a deprecation warning to them (like we do [here](https://github.com/seladb/PcapPlusPlus/blob/22c6c85130ececa0cd4f9367332b8142687722c4/Common%2B%2B/header/IpAddress.h#L151)) and add 2 new properties:
```cpp
timespec startTimestamp;
timespec endTimestamp;
```